.mk/development.mk

```diff
@@ -69,3 +69,23 @@ undeploy-infra: undeploy-loki undeploy-grafana uninstall
 
 .PHONY: undeploy-all
 undeploy-all: undeploy-infra undeploy-sample-cr undeploy-sample-workload
+
+.PHONY: deploy-prometheus
+deploy-prometheus: ## Deploy prometheus.
+	@echo -e "\n==> Deploy prometheus"
+	@echo -e "\n==> Obtain flp node addr/ports"
+	pod_list=$$(kubectl get pods --selector=app=flowlogs-pipeline -o jsonpath='{.items[*].status.podIP}'); \
+	FLP_TARGET_LIST=$$(echo $$pod_list | sed 's~^~[~; s~ ~:9102,~g; s~$$~:9102]~;') ; \
+	sed "s~%FLP_TARGETS%~$$FLP_TARGET_LIST~" config/kubernetes/deployment-prometheus.yaml >/tmp/deployment-prometheus.yaml
+	kubectl apply -f /tmp/deployment-prometheus.yaml
+	kubectl rollout status "deploy/prometheus" --timeout=600s
+	-pkill --oldest --full "9090:9090"
+	kubectl port-forward --address 0.0.0.0 svc/prometheus 9090:9090 2>&1 >/dev/null &
```

Comment:
can you move this port-forward into an identical condition as we did for loki and grafana:

```
ifeq (true, $(PORT_FWD))
... do port forward
endif
```